File: profile_service/src/main/java/huy/project/profile_service/core/domain/dto/request/BaseParams.java
package huy.project.profile_service.core.domain.dto.request;

import lombok.*;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class BaseParams {
    private Integer page;
    private Integer pageSize;
    private String sortBy;
    private String sortType;
}

File: promotion_service/PromotionService.Core/Domain/Dto/Request/BaseParams.cs
namespace PromotionService.Core.Domain.Dto.Request;

public class BaseParams
{
    public int Page { get; set; } = 0;
    public int PageSize { get; set; } = 10;
    public string SortBy { get; set; } = "Id";
    public string SortOrder { get; set; } = "asc";
}